src/eval-frame/actions/actions.js

```diff
@@ -17,6 +17,36 @@ import { postMessageToEditor } from "../port-to-editor";
 const MD = MarkdownIt({ html: true });
 MD.use(MarkdownItKatex).use(MarkdownItAnchor);
 
+/* eslint-disable */
```

Comment:
taken from http://tobyho.com/2012/07/27/taking-over-console-log/ - if we are going to take this seriously we can refactor this so it makes more sense.